gvm/protocols/gmpv8.py

```diff
@@ -541,3 +546,219 @@ def modify_tag(
                 _xmlresources.add_element("type", resource_type)
 
         return self._send_xml_command(cmd)
+
+    def clone_ticket(self, ticket_id):
+        """Clone an existing ticket
+
+        Arguments:
+            result_id (uuid): Result the ticket applies to
+            assigned_to_user_id (uuid): User the ticket is assigned to
+            open_note (str): A note about opening the ticket
+            comment (str, optional): Comment for the ticket
+
```

Comment:
On a closer look, the last two arguments `open_note` and `comment` either do not exist or are not implemented. Did not notice that in my earlier suggestion ...

Reply:
You are right. Was a copy&paste mistake.